Android/Guava/samples/GuavaSample/Future.cs

```diff
@@ -0,0 +1,25 @@
+﻿using Google.Common.Util.Concurrent;
+using Java.Lang;
+using Java.Util.Concurrent;
+
+namespace GuavaSample
+{
+    // Just ensure this compiles
+    public class Future : Java.Lang.Object, IListenableFuture
```

Comment:
This sample isn't really that helpful, basically just an "integration test" checking a type exists.

When I look at `GuavaSample.dll` it has the same `.jar` files embedded as before:

![image](https://user-images.githubusercontent.com/840039/119562435-f258fa80-bd6b-11eb-9654-fa485799e0c1.png)